src/loaders/config/mod.rs

```diff
@@ -0,0 +1,55 @@
+use std::fs;
+
+use ini::Ini;
+
+#[cfg(feature = "debug")]
+use crate::print_debug;
+#[cfg(feature = "debug")]
+use crate::Timer;
+#[cfg(feature = "debug")]
+use crate::{CYAN, MAGENTA, NONE};
+
+#[derive(Default)]
+pub struct ConfigLoader {
+    pub game_files_list: Vec<String>,
+}
+
+pub const GAME_FILES_LIST_PATH: &str = "DATA.INI";
+
+impl ConfigLoader {
+    pub fn load_game_file_list(&mut self) {
+        let mut game_files_list = Vec::new();
+
+        #[cfg(feature = "debug")]
+        let timer = Timer::new_dynamic(format!("load game data from {MAGENTA}{GAME_FILES_LIST_PATH}{NONE}"));
+        let game_files_list_file = Ini::load_from_file(GAME_FILES_LIST_PATH);
+
+        if game_files_list_file.is_ok() {
+            let ini_file = game_files_list_file.unwrap();
+            let data_section = ini_file
+                .section(Some("data"))
+                .expect(&format!("Failed to read DATA section from {GAME_FILES_LIST_PATH}"));
+
+            #[cfg(feature = "debug")]
+            timer.stop();
+
+            for (order, path) in data_section.iter() {
+                #[cfg(feature = "debug")]
+                print_debug!(format!(
+                    "reading from order {MAGENTA}{order}{NONE} the game file from path {CYAN}{path}{NONE}"
+                ));
+
+                let index = order.parse::<usize>().unwrap();
+                game_files_list.insert(index, path.trim().to_string());
+            }
+        } else {
+            game_files_list_file.expect(&format!("Failed to read {GAME_FILES_LIST_PATH}"));
+        }
```

Comment:
This can actually be expressed very nicely in Rust using a `let-else` statement. The code would then look like
```rust
        let Ok(ini_file) = game_files_list_file else {
            panic!("failed to read {MAGENTA}{GAME_FILES_LIST_PATH}{NONE}");
        };

        // continue working with `ini_file`
        // ..
```
The advantage here is that you don't need to unwrap or check for `is_ok()`, but can just model the happy path.

Note that I usually start panic messages with a lowercase letter